obsplus/events/get_events.py

```diff
@@ -145,6 +146,22 @@ def _get_ids(df, kwargs) -> set:
     return set(df.event_id[:limit])
 
 
+def _handle_dateline_transversal(filt, df, kwargs):
+    """Check if dateline should be transversed by query."""
+    # if longitudes aren't being used bail out
+    if not {"minlongitude", "maxlongitude"}.issubset(set(kwargs)):
+        return filt, kwargs
+    # if dateline is not to be transversed by query bail out
+    minlong, maxlong = kwargs["minlongitude"], kwargs["maxlongitude"]
+    if not minlong > maxlong:
```

Comment:
I think when `maxlongitude` is given incorrectly as > 180 this ignores the dateline issue.

Reply:
You are right, I forced the longitude values to be in the expected range in other parts of the code but not for the input to these queries. Luckily I made a nice little utility for this already, so it was an easy fix. It should work as expected now.